website_sale_product_description_top/views/website_sale_template.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="product_remove" inherit_id="website_sale_product_description.product">
```

Comment:
I read in the Odoo book that replacing is not recommended for removing elements (I don't know about 14.0 though). Maybe `position="move"` should be preferred ?

Reply:
:open_mouth: 

You're exactly right. `position="move"` seems like it's the correct thing.